src/cephadm/tox.ini

```diff
@@ -62,4 +62,4 @@ deps =
     flake8-quotes
 commands =
     flake8 --config=tox.ini {posargs:cephadm}
-    bash -c "test $(grep 'docker.io' cephadm | wc -l) == 11"
+    bash -c "test $(grep 'docker.io' cephadm | wc -l) == 9"
```

Comment:
I wonder what the purpose of this test is.

Reply:
Sebastian wanted to keep track of the number of docker.io references we make https://github.com/ceph/ceph/pull/44306/files#r769453080

Reply:
Thanks for that reference!

I've added a comment explaining this in https://github.com/ceph/ceph/pull/45056. I can rebase it after we merge this PR.